envbuilder.go

```diff
@@ -194,6 +194,11 @@ type Options struct {
 	// This is useful for self-signed certificates.
 	SSLCertBase64 string `env:"SSL_CERT_BASE64"`
 
+	// ExportEnvFile is an optional file path where envbuilder
+	// will dump environment variables from devcontainer.json
+	// and the built container image.
+	ExportEnvFile string `env:"EXPORT_ENV_FILE"`
```

Comment:
A comment describing the use case would also be helpful, mainly because of the `export` directive, which is technically `sh` syntax, not `.env` file.

Reply:
I wonder if we should make this a proper `.env` file being exported instead. Would that make it less useful?